examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2335,9 +2335,7 @@
                   "title": "Repeated numeric enum title"
                 },
                 "enumValueAnnotation": {
-                  "$ref": "#/definitions/examplepbNumericEnum",
-                  "description": "Numeric enum description.",
-                  "title": "Numeric enum title"
```

Comment:
I might have asked this already, but do we understand why these were removed?

Reply:
It's explained [in the code](https://github.com/grpc-ecosystem/grpc-gateway/pull/2904/files#diff-c904837bc1a47dae3fba9863430f2ab46fa8a1bd0c9e25475308ab426a0d4efeR520-R521) (even though I personally would like to have descriptions next to $ref:s):
```
// Per the JSON Reference syntax: Any members other than "$ref" in a JSON Reference object SHALL be ignored.
// https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03#section-3
```
More info here:
https://swagger.io/docs/specification/using-ref/

Reply:
Ah, so these appearing here was already a bug?

Reply:
Yes, or they are ignored so it's unnecessary to render them there I guess.